pkg/releasesync/releasesync.go

```diff
@@ -191,12 +191,30 @@ func releaseDetailsToDB(architecture string, tag ReleaseTag, details ReleaseDeta
 		return nil // changelog not available yet
 	}
 
+	// not ready to enable this yet
+	// presume we must have some components, kubernetes, coreos
+	// if len(details.ChangeLogJSON.Components) > 0 {
+	// 	jsonChangeLog := parseChangeLogJSON(tag.Name, details.ChangeLogJSON)
+	//
+	// 	release.KubernetesVersion = jsonChangeLog.KubernetesVersion
+	// 	release.CurrentOSURL = jsonChangeLog.CurrentOSURL
+	// 	release.CurrentOSVersion = jsonChangeLog.CurrentOSVersion
+	// 	release.PreviousOSURL = jsonChangeLog.PreviousOSURL
+	// 	release.PreviousOSVersion = jsonChangeLog.PreviousOSVersion
+	// 	release.OSDiffURL = jsonChangeLog.OSDiffURL
+	//
+	// 	release.PreviousReleaseTag = jsonChangeLog.PreviousReleaseTag
+	// 	release.Repositories = jsonChangeLog.Repositories
+	// 	release.PullRequests = jsonChangeLog.PullRequests
+	//
+	// } else {
```

Comment:
I would suggest writing this in a way that it will just work without needing to uncomment the code.  You should be able to check if ChangeLogJSON is populated, and if so use that.